quesma/clickhouse/quesma_communicator.go

```diff
@@ -160,7 +160,6 @@ func executeQuery(ctx context.Context, lm *LogManager, query *model.Query, field
 	//
 
 	settings := make(clickhouse.Settings)
-	settings["readonly"] = "1"
```

Comment:
Why is that?

Reply:
Apparently that causes troubles with Hydrolix queries - I'll link you some stack traces.